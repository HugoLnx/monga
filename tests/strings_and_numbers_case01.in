99 30 123 "hugo roque"
90
"hey"
