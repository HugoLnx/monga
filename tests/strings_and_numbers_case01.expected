NUMBER,99
NUMBER,0
TEXT,"hugo roque 938z"
NUMBER,-50
TEXT,"hey\n\t\"\^\~\-\+"
TEXT,""
TEXT,"\\"
TEXT,"\" very evil sample \""
HEXADECIMAL,1
HEXADECIMAL,0
HEXADECIMAL,256
HEXADECIMAL,-32
HEXADECIMAL,2560
HEXADECIMAL,240
HEXADECIMAL,-240
CHAR,'a'
CHAR,'\n'
CHAR,'\''
CHAR,'"'
CHAR,'\t'
CHAR,'\\'
TEXT,"teste \n"
